versions/3.0.md

```diff
@@ -972,25 +972,25 @@ Field Name | Type | Description
 <a name="parameterAllowEmptyValue"></a> allowEmptyValue | `boolean` | Sets the ability to pass empty-valued parameters. This is valid only for `query` parameters and allows sending a parameter with an empty value. Default value is `false`. If [`style`](#parameterStyle) is used, if behavior is `n/a`, the value of `allowEmptyValue` SHALL be ignored.
 
 The rules for serialization of the parameter are specified in one of two ways.
-For simpler scenarios, a [`schema`](#parameterSchema) and [`style`](#parameterStyle) can be used to describe the structure and syntax of the parameter.
+For simplicity, a [`schema`](#parameterSchema) and [`style`](#parameterStyle) can describe the structure and syntax of the parameter.
```

Comment:
No, it was the 'simpler scenarios' I preferred, it indicates there are **two** different scenarios, a simpler one and one described in the subsequent paragraph ('For more complex scenarios...'), rather than a simple and complex way of doing the same thing. It is effectively a run-on sentence with a table in the middle of it. The 'complex scenario' does still need an example.